### ISSUE TITLE
`translateAttr` is broken with ember-2.9-beta.1

### ISSUE THREAD

**Issue by workmanw** [jamesarosen/ember-i18n#402]
I get this cryptic stack on one of my routes with ember-2.9-beta.1. I removed `translateAttr` and it restored functionality. It's not at all clear to me why this is caused by `translateAttr`. The only reason I figured it out was by chance. 

```
on-error.js:77 TypeError: Cannot read property 'indexOf' of undefined
    at Object.unregisterAction (ember.debug.js:10063)
    at ActionState.destroy (ember.debug.js:10179)
    at SimpleBlockTracker.destroy (ember.debug.js:42304)
    at SimpleBlockTracker.destroy (ember.debug.js:42304)
    at SimpleBlockTracker.destroy (ember.debug.js:42304)
    at UpdatableBlockTracker.destroy (ember.debug.js:42304)
    at SimpleBlockTracker.destroy (ember.debug.js:42304)
    at SimpleBlockTracker.destroy (ember.debug.js:42304)
    at UpdatableBlockTracker.destroy (ember.debug.js:42304)
    at SimpleBlockTracker.destroy (ember.debug.js:42304)
```

**Comment by rwjblue** [comment]
This feels like an Ember bug to me.  Can you throw up a repo that has a failing test that I can dig into?

**Comment by workmanw** [comment]
Hmm. Well as I creating a reproduction for you, I discovered that `translateAttr` is actually no longer supported by ember-i18n at all. 😳. Apparently we missed a few of them when we upgraded last time and they just didn't error. Sorry for the false alarm here :(. 

@rwjblue I'm going to close this issue. I tried to actually reproduce this issue in a repo (missing a helper) and it seems to give me a sane meaningful error: `Compile Error: translateAttr is not a modifier`. I'm not sure why I get a cryptic stack in this one case.
